### PR TITLE
Separate `postulate`s from statements for function extensionality and univalence

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Moves function extensionality and univalence postulates from `foundation-core` to `foundation`. Builds on #400.

This is a fixed version of #401.

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
I do in fact think this is a useful pull request. In my own formalizations, I want to use your library without necessarily invoking univalence. By being more sanitary with when the axioms are invoked it improves the library's usability for me.

File: src/foundation-core/function-extensionality.lagda.md [hunk 1]
```diff
@@ -90,3 +53,9 @@ ap-ev :
   (p : f ＝ g) → (ap (λ h → h a) p) ＝ htpy-eq p a
 ap-ev a refl = refl
 ```
+ 
+ ## See also
+ 
+ - That the univalence axiom implies function extensionality is proven in
+   [foundation.univalence-implies-function-extensionality](foundation.univalence-implies-function-extensionality.html).
+- Weak function extensionality is defined in [foundation.weak-function-extensionality](foundation.weak-function-extensionality.html).
```
Comment (EgbertRijke) [inline src/foundation-core/function-extensionality.lagda.md:61]: It seems that the indentation is off here.

File: src/foundation/function-extensionality.lagda.md [hunk 2]
```diff
@@ -8,4 +8,62 @@ title: Function extensionality
...
+  equiv-eq-htpy : {f g : (x : A) → B x} → (f ~ g) ≃ (f ＝ g)
+  pr1 (equiv-eq-htpy {f = f} {g}) = eq-htpy
+  pr2 (equiv-eq-htpy {f = f} {g}) = is-equiv-eq-htpy f g
 ```
+
+ ## See also
+ 
+ - That the univalence axiom implies function extensionality is proven in
+   [foundation.univalence-implies-function-extensionality](foundation.univalence-implies-function-extensionality.html).
+- Weak function extensionality is defined in
```
Comment (EgbertRijke) [inline src/foundation/function-extensionality.lagda.md:68]: The indentation is off here again.

Comment (fredrik-bakke) [reply src/foundation/function-extensionality.lagda.md:68]: How do you want it?

Comment (fredrik-bakke) [reply src/foundation/function-extensionality.lagda.md:68]: two spaces or all on one line?

**Comment by EgbertRijke** [comment]
Thanks! I'll merge it when the checks are done.

**Comment by fredrik-bakke** [comment]
And sorry to bother you with these menial PRs. At least now it's done and I can get on with my actual contributions! 😄

**Comment by EgbertRijke** [comment]
No need to apologize! You made great improvements to the library.

If you're on Discord we can discuss there how your other work can be integrated in the library. We can also discuss over email, if you prefer.

**Comment by fredrik-bakke** [comment]
Sure, I'll send a message on discord when I'm ready. Maybe I'm just weird, but I find it difficult to communicate in channels with many participants.

**Pull request merged**